ts/utils/dates.ts

```diff
@@ -68,18 +71,54 @@ export function format(
   );
 }
 
-export function isExpired(expireMonth: number, expireYear: number): boolean {
-  const today: Date = new Date();
-  const currentMonth: number = today.getMonth() + 1;
-  const currentYear: number = parseInt(
-    today.getFullYear().toString().slice(2),
-    10
-  );
-  return (
-    expireYear < currentYear ||
-    (expireYear === currentYear && expireMonth < currentMonth)
-  );
-}
+export const dateFromMonthAndYear = (
+  month: string | number | undefined,
+  year: string | number | undefined
+): Option<Date> => {
```

Comment:
Maybe the return type of this function should be `Either<Error, Date>`, just to reflect that with an incorrect input value, supplied to the method, an error occurs. In this way could be possible also add additional information about the reason of the failure (could be an useful information sometime?). What do you think?

Reply:
https://github.com/pagopa/io-app/pull/2982#discussion_r616576915